applications/StructuralMechanicsApplication/python_scripts/sprism_process.py

```diff
@@ -41,12 +39,14 @@ def __init__(self, Model, settings):
         {
             "mesh_id"                        : 0,
             "model_part_name"                : "Structure",
-            "preprocess_shell_to_solidshell" : {
+            "preprocess_shell_to_solidshell" : false,
```

Comment:
false by default?

Reply:
yes, indded the error you notice yesterday was because of this, if the elements already exist this is wrong